mrt/static/js/printouts.js

```diff
@@ -21,7 +21,13 @@ $(function () {
         var job_row = $('#' + this.id);
         job_row.addClass('success');
         job_row.find('.status').text(STATUS.FINISHED);
-        var html = $('<b>').text('Success - please refresh');
+        
+        var filePath = "[?:[a-zA-Z0-9-_\.]+(?:.xls|.xlsx)";
```

Comment:
This will fail to match for pdf files, also it will fail to match filenames with other characters (e.g. spaces). 

I suggest just checking if it's a link (either absolute or relative) instead of checking if it's a specific file. Should make it easier to maintain whenever new download types are added, and make the code more resilient.